### ISSUE TITLE
mqtt in webGL, can not connect to the broker

### ISSUE THREAD

**Issue by SoftwareMechanic** [vovacooper/Unity3d_MQTT#10]
Hello, I can not connect to the broker when launch my project using webGL, i tried using websockets ( that worked for me in a different javascript project which obviously runs in the browser) , I also tried to use unityScript (.jslib) to create a connection with the server which hosts the broker, but didn't work.
can you please help me ???

**Comment by sm-j** [comment]
Did you find any solution to this WebGL issue?

**Comment by SoftwareMechanic** [comment]
@sm-j Yes, the solution is to set up the mqtt in JavaScript and then call unity functions by JavaScript ( gameInstance.SendMessage()
